src/Components/Generator/ActionButton.js

```diff
@@ -5,8 +5,9 @@ import MailtoDropdown from '../MailtoDropdown';
 
 export default class ActionButton extends preact.Component {
     render(props) {
+        const enabled = props.transport_medium === 'email' ? !!props.letter : props.download_active;
```

Comment:
If you create that `const` you can just as well use it further down.

Reply:
Done.